packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -204,23 +228,47 @@ portfolio.notifyOnStackEvents(product, topic2, {
 });
 ```
 
-### Tag update constraint
+### Set launch role
 
-Tag update constraints allow or disallow end users to update tags on resources associated with an AWS Service Catalog product upon provisioning.
-By default, tag updating is not permitted.
-If tag updating is allowed, then new tags associated with the product or portfolio will be applied to provisioned resources during a provisioned product update.
+Allows you to configure a specific AWS `IAM` role that a user must assume when launching a product.
+By setting this launch role, you can control what policies and privileges end users can have.
+The launch role must be assumed by the service catalog principal.
+You can only have one launch role set for a portfolio-product association, and you cannot set a launch role if a StackSets deployment has been configured.
 
 ```ts fixture=portfolio-product
-portfolio.addProduct(product);
+const launchRole = new iam.Role(this, 'LaunchRole', {
+  assumedBy: new iam.ServicePrincipal('servicecatalog.amazonaws.com'),
+});
 
-portfolio.constrainTagUpdates(product);
+portfolio.setLaunchRole(product, launchRole);
 ```
 
-If you want to disable this feature later on, you can update it by setting the "allow" parameter to `false`:
+See [Launch Constraint](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints-launch.html) documentation
+to understand permissions roles need.
+
+### Deploy with StackSets
+
+A StackSets deployment constraint allows you to configure product deployment options using 
+[AWS CloudFormation StackSets](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/using-stacksets.html). 
+You can specify multiple accounts and regions for the product launch following StackSets conventions.
+There is an additional field `allowStackControl` that configures ability for end users to create, edit, or delete the stacks. 
+End users can manage those accounts and determine where products deploy and the order of deployment.
+You can only define one StackSets deployment configuration per portfolio-product association,
+and you cannot both set a launch role and StackSets deployment configuration for an assocation.
 
 ```ts fixture=portfolio-product
-// to disable tag updates:
-portfolio.constrainTagUpdates(product, {
-  allow: false,
+const adminRole = new iam.Role(this, 'AdminRole', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+// The execution role deployed in target accounts
+const executionRole = iam.Role.fromRoleArn(this, 'ImportedExecutionRole', 'arn:aws:iam::account:role/StackSetExecutionRole');
```

Comment:
What does it mean that you're using `account` here, but the `accounts` property is `['012345678901', '012345678902', '012345678903']`?

Reply:
this is why I realized using `executionRoleName` might make more sense, we need the name for the role thats already in the accounts.  In another app/stack where those are defined we might have a constructor that creates this role in all accounts but there isn't a way to cleanly/clearly 'import' that here.  I just used `account` since it's a wildcard, but would have maybe made more sense to just use one of the account numbers in the `accounts` arg.

Reply:
Yep. Let's switch to a `string` for the execution Role.

Reply:
I went with naming it `executionRoleName`, which immediately gives more info, but also just having 2 args with `*Role` being different types does not seem ideal.